virt/virt-2-6-release-notes.adoc

```diff
@@ -32,10 +32,26 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-7544: Microsoft's Windows Server Virtualization Validation Program (SVVP) applies to RHCOS workers
 
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
+
+** Intel and AMD CPUs.
+
 //CNV-9723: VMs are now migrated when a node is drained
 
 //CNV-9521: VMs that use UEFI can now be booted by using secure boot
 
+* You can now boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode.
++
+[NOTE]
+====
+{VirtProductName} only supports a VM with Secure Boot when using EFI mode. If Secure Boot is not enabled, the VM crashes repeatedly. However, the VM might not support Secure Boot. Before you boot a VM, verify that it supports Secure Boot by checking the VM settings.
+====
+
+Learn more about xref:../virt/virtual_machines/advanced_vm_management/virt-efi-mode-for-vms.adoc#virt-efi-mode-for-vms[VMs and EFI Mode].
```

Comment:
@mgarrellRH I would suggest you add the xref as part of the intro sentence. "You can now _boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode_." where "boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode" becomes the link text.